ccd-definition/CaseEvent/PREPARE_FOR_HEARING.json

```diff
@@ -200,5 +200,33 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/comply-with-directions/about-to-start",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/comply-with-directions/about-to-submit",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "limitedUploadDocuments-PREPARE_FOR_HEARING",
+    "Name": "Documents",
+    "Description": "Upload documents",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "PREPARE_FOR_HEARING",
+    "PostConditionState": "PREPARE_FOR_HEARING",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "addStatementOfService-PREPARE_FOR_HEARING",
+    "Name": "Add statement of service (c9)",
+    "Description": "Add statement of service",
+    "DisplayOrder": 13,
```

Comment:
I think display order with those two new elements is a bit off.

Reply:
done